libs/api/domains/regulations-admin/src/lib/graphql/models/draftRegulation.model.ts

```diff
@@ -91,9 +91,6 @@ export class DraftRegulationModel {
   @Field(() => Date, { nullable: true })
   idealPublishDate?: string
 
-  @Field(() => Date, { nullable: true })
-  ministryId?: MinistrySlug
-
```

Comment:
Hmm... why delete? Shouldn't this be just renamed and changed to `string` type like in the other files above?

Reply:
This is just invalid stuff, there is another ministry string in the file